app/assets/javascripts/admin/addon/models/web-hook.js

```diff
@@ -1,13 +1,32 @@
-import { computed } from "@ember/object";
+import EmberObject, { computed } from "@ember/object";
 import { isEmpty } from "@ember/utils";
 import { observes } from "@ember-decorators/object";
-import Category from "discourse/models/category";
 import Group from "discourse/models/group";
 import RestModel from "discourse/models/rest";
 import Site from "discourse/models/site";
 import discourseComputed from "discourse-common/utils/decorators";
 
+class WebHookExtras extends EmberObject {
+  constructor(args) {
+    super(...arguments);
+
+    this.categories = args.categories;
```

Comment:
Right now you're extending from EmberObject, so this assigment would happen automatically in the `super()`.

However... given that we're trying to move away from EmberObject, how about keeping this manual assignment and dropping the `extends EmberObject`? We can add `@tracked` and drop `@computed` while we're at it.

Something like

```js
class WebHookExtras {
  @tracked categories;

  constructor(args){
    this.categories = args.categories;
  }

  get categoriesById(){
    if (this.categories) {
      return new Map(this.categories.map((c) => [c.id, c]));
    }
  }
}
```